src/components/chat-message.tsx

```diff
@@ -158,9 +160,12 @@ export const ChatMessage = ( {
 				role="group"
 				aria-labelledby={ id }
 				className={ cx(
-					'inline-block p-3 rounded border border-gray-300 overflow-x-auto select-text',
+					'inline-block p-3 rounded overflow-x-auto select-text',
 					isUnauthenticated ? 'lg:max-w-[90%]' : 'lg:max-w-[70%]', // Apply different max-width for unauthenticated view
-					! isUser ? 'bg-white' : 'bg-white/45'
+					! isUser ? 'bg-white' : '',
+					isUser && ! hasError ? 'bg-white/45' : '',
+					hasError ? 'border-[#FACFD2] bg-[#F7EBEC]' : 'border-gray-300',
+					'border'
```

Comment:
Do we need to place this class name at the end of the argument list? If not, probably we could move it to the first item.